script/install-as-user

```diff
@@ -158,6 +164,7 @@ then
         -e "s,^( *CONTACT_EMAIL:).*,\\1 'postmaster@$HOST'," \
         -e "s,^( *TRACK_SENDER_EMAIL:).*,\\1 'postmaster@$HOST'," \
         -e "s,^( *SECRET_KEY_BASE:).*,\\1 '$RANDOM_COOKIE_SECRET'," \
+        -e "s,^( *STAGING_SITE:).*,\\1 '$STAGING_SITE'," \
```

Comment:
Isn't the intention that `STAGING_SITE` is set in `general.yml`. AIUI, this would overwrite that value?

Or are we saying that irrespective of the `general.yml` configuration, `STAGING_SITE` should be set to `1` if `DEVELOPMENT_INSTALL` is `true`? This seems a bit confusing in that the expected config would be ignored. Could it error out instead (with a message like "Incompatible config options! `STAGING_SITE` must be `1` for a `DEVELOPMENT_INSTALL`"?)

Reply:
This only gets run when `general.yml` doesn't exist so is setting the default value based on if the install script has the `--development` flag or not.

Reply:
If the install script is rerun then yes maybe we should warn of "Incompatible config options". Would you like me to add that now?

Reply:
Ah, yeah, sounds good then.

Reply:
> "Incompatible config options". Would you like me to add that now?

If it's a quick change it would likely avoid confusion.

Reply:
See: https://github.com/mysociety/alaveteli/pull/7622